cpp/src/tsne/exact_tsne.cuh

```diff
@@ -51,6 +52,7 @@ void Exact_TSNE(value_t *VAL, const value_idx *COL, const value_idx *ROW,
                 const value_idx NNZ, const raft::handle_t &handle, value_t *Y,
                 const value_idx n, const value_idx dim,
                 const float early_exaggeration = 12.0f,
+                const float late_exaggeration = 1.0f,
```

Comment:
This hasn't really been tested in the exact tsne either. I think we could remove this and add to the issue for the barnes-hut. It's easy enough to add this back in using FIT-SNE as an example.